content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -290,73 +292,73 @@ This returns metrics as follows:
-# TYPE pool_wait_count gauge
-pool_wait_count 0
-
-# HELP query_active_transactions
-# TYPE query_active_transactions gauge
-query_active_transactions 0
-
-# HELP pool_wait_duration_ms The wait time for a worker to get a connection.
-# TYPE pool_wait_duration_ms histogram
-pool_wait_duration_ms_bucket{le="0"} 0
-pool_wait_duration_ms_bucket{le="1"} 3
-pool_wait_duration_ms_bucket{le="5"} 3
-pool_wait_duration_ms_bucket{le="10"} 3
-pool_wait_duration_ms_bucket{le="50"} 3
-pool_wait_duration_ms_bucket{le="100"} 3
-pool_wait_duration_ms_bucket{le="500"} 3
-pool_wait_duration_ms_bucket{le="1000"} 3
-pool_wait_duration_ms_bucket{le="5000"} 3
-pool_wait_duration_ms_bucket{le="50000"} 3
-pool_wait_duration_ms_bucket{le="+Inf"} 3
-pool_wait_duration_ms_sum 0.023208
-pool_wait_duration_ms_count 3
-
-# HELP query_operation_total_elapsed_time_ms
-# TYPE query_operation_total_elapsed_time_ms histogram
-query_operation_total_elapsed_time_ms_bucket{le="0"} 0
-query_operation_total_elapsed_time_ms_bucket{le="1"} 1
-query_operation_total_elapsed_time_ms_bucket{le="5"} 2
-query_operation_total_elapsed_time_ms_bucket{le="10"} 2
-query_operation_total_elapsed_time_ms_bucket{le="50"} 2
-query_operation_total_elapsed_time_ms_bucket{le="100"} 2
-query_operation_total_elapsed_time_ms_bucket{le="500"} 2
-query_operation_total_elapsed_time_ms_bucket{le="1000"} 2
-query_operation_total_elapsed_time_ms_bucket{le="5000"} 2
-query_operation_total_elapsed_time_ms_bucket{le="50000"} 2
-query_operation_total_elapsed_time_ms_bucket{le="+Inf"} 2
-query_operation_total_elapsed_time_ms_sum 3.197624
-query_operation_total_elapsed_time_ms_count 2
-
-# HELP query_total_elapsed_time_ms
-# TYPE query_total_elapsed_time_ms histogram
-query_total_elapsed_time_ms_bucket{le="0"} 0
-query_total_elapsed_time_ms_bucket{le="1"} 5
-query_total_elapsed_time_ms_bucket{le="5"} 5
-query_total_elapsed_time_ms_bucket{le="10"} 5
-query_total_elapsed_time_ms_bucket{le="50"} 5
-query_total_elapsed_time_ms_bucket{le="100"} 5
-query_total_elapsed_time_ms_bucket{le="500"} 5
-query_total_elapsed_time_ms_bucket{le="1000"} 5
-query_total_elapsed_time_ms_bucket{le="5000"} 5
-query_total_elapsed_time_ms_bucket{le="50000"} 5
-query_total_elapsed_time_ms_bucket{le="+Inf"} 5
-query_total_elapsed_time_ms_sum 1.8407059999999997
-query_total_elapsed_time_ms_count 5
+# HELP prisma_datasource_queries_total
+# TYPE prisma_datasource_queries_total counter
+prisma_datasource_queries_total 5
+
+# HELP prisma_pool_connections_busy The number of active connections in use.
+# TYPE prisma_pool_connections_busy gauge
+prisma_pool_connections_busy 0
+
+# HELP prisma_pool_connections_idle The number of connections that are not being used
+# TYPE prisma_pool_connections_idle gauge
+prisma_pool_connections_idle 21
+
+# HELP prisma_client_queries_wait The number of workers waiting for a connection.
+# TYPE prisma_client_queries_wait gauge
+prisma_client_queries_wait 0
+
+# HELP prisma_client_queries_active
```

Comment:
This seems to be missing a definition

Reply:
For what it worth, description is missing in JSON format in current version of the client as well. Should we add it @garrensmith?